app/src/main/java/com/jerboa/ui/components/post/create/CreatePostActivity.kt

```diff
@@ -52,7 +53,7 @@ data class MetaDataRes(val title: String?, val loading: Boolean)
 
 @Composable
 fun CreatePostActivity(
-    accountViewModel: AccountViewModel,
+    accountViewModel: AccountViewModel = viewModel(factory = AccountSettingsViewModelFactory.Factory),
```

Comment:
With regards to all of these, I think I'd much prefer them not having defaults, and filling this from MainActivity. Its much more explicit that way, and I've tried to avoid defaults, especially for viewmodels, wherever possible.